### PR TITLE
Feature | Multi Sim Uses new tracer

### PR DESCRIPTION
**Pull request by alexcampbelling**

### CODE REVIEW

File: core/state_processor.go [hunk 1]
```diff
@@ -17,25 +17,28 @@
...
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/eth/tracers/logger"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/params"
-	"github.com/holiman/uint256"
 )
 
+// "github.com/ethereum/go-ethereum/eth/tracers/native/txnOpCodeTracer"
```
Comment (dmarzzz) [inline core/state_processor.go:40]: remove pl0x

File: core/state_processor.go [hunk 2]
```diff
@@ -199,251 +202,166 @@ func ApplyTransaction(config *params.ChainConfig, bc ChainContext, author *commo
...
-				storage[fmt.Sprintf("%x", i)] = fmt.Sprintf("%x", storageValue)
-			}
-			formatted[index].Storage = &storage
-		}
-	}
-	return formatted
+// The below is a copy from "eth/tracers/native/txnOpCodeTracer.go" to solve  acurrent circular dependency, this is tech debt to remove
+// In the future we would like to call something similar to "tracer, err := tracers.New("newtxnOpCodeTracer", nil, nil)"
+
+type callFrameBN struct {
```
Comment (dmarzzz) [inline core/state_processor.go:216]: could you put this in the common package , can just move to a new file `./common/bn_tracer.go` and then import like `common.callFrameBN`

File: internal/ethapi/api.go [hunk 3]
```diff
@@ -2063,28 +2063,33 @@ type CallBundleArgs struct {
 	BaseFee    *big.Int          `json:"baseFee"`
 }
 
-//
-// CallBundle will simulate a bundle of transactions on top of
-// the most recent block. Partially follows flashbots spec v0.5.
+// TODO: this is a rewrite of the below CallBundle API, I am attempting to recreate the logic but skip on all
```
Comment (dmarzzz) [inline internal/ethapi/api.go:2066]: Great comment adds in this function! Could we revert this top level message back to what it was before to give flashbots attribution + references the spec version

**Review by dmarzzz — CHANGES_REQUESTED** [review]
Small change requests , nothing big. Also I understand the approach has duplicate code due to some tech debt so no worries there

File: core/state_processor.go (hunk 2 above)
Comment (alexcampbelling) [reply core/state_processor.go:214]: smart, will do this now and run another test before I come back

Comment (alexcampbelling) [reply core/state_processor.go:214]: Could not do this cleanly without creating errors for import cycles. Decided against this as only including `callFrameBN` is probably not important to abstract out but leave the rets of the hard coded tracer in `state_processor.go` anyhow. 

I am thinking through a solution to get all of this out and clean, but that scope is out of this effort.

**Review by dmarzzz — APPROVED** [review]

**Pull request merged**